activejob/lib/active_job/exceptions.rb

```diff
@@ -151,7 +151,8 @@ def determine_delay(seconds_or_duration_or_algorithm:, executions:, jitter: JITT
 
       def determine_jitter_for_delay(delay, jitter)
         return 0.0 if jitter.zero?
-        Kernel.rand(delay * jitter)
+        jitter_multiplier = delay * jitter.to_f
+        Kernel.rand(0..jitter_multiplier.abs)
```

Comment:
Passed a range instead of a single number. With a positive float range, the return value will be a random float number from the range.

Reply:
Do we need to `#abs` the total jitter?

TIL that Kernel rand accepts ranges! Cool.

A simple, clear option, since Kernel.rand always returns a 0-1 float:
```ruby
Kernel.rand * delay * jitter
```

Reply:
@jeremy Thanks for your kind words in the earlier comment! I'm glad I can be of help! About the comments here,

I used `#abs` because `Kernel.rand(0..any_negative_value)` results in `nil`. And the reason for using range was to make sure the implementation was consistent with earlier test cases wherein we were stubbing `Kernel.rand` and verifying values (especially when verifying that the default value of ActiveJob::Base.retry_jitter is used).
I'll make changes as suggested and will try to update the test cases. Thanks again for your review!

Reply:
@jeremy Updated the PR with suggested changes. Has made it pretty simple, readable and arguably faster! Thanks for the suggestion.

I'm still in dilemma about using `abs` for provided jitter. Although it is up to the user to decide the jitter to add and its expected results (for instance negative resultant wait time in an extreme case when negative jitter value is provided). But maybe I'm overthinking this.  😄 

To make sure the basic case works fine, I've added a test case to verify a randomized wait time with negative jitter value provided.

Reply:
Jitter is meant to be interpreted as a percentage, so negative values are out of bounds. I think it's safe to ignore that case for this PR. (If anything, we whould `raise ArgumentError if jitter < 0`, I think.)